src/test/lambda/commands/deleteLambda.test.ts

```diff
@@ -76,29 +81,47 @@ describe('deleteLambda', async () => {
             async configuration => assert.fail()
         )
 
-        await deleteLambda(node, () => {})
+        await deleteLambda({
+             node,
+             lambdaClient,
+             outputChannel: new MockOutputChannel(),
+             onRefresh: () => refreshCount++,
+             onConfirm: async () => confirmationResponse
+        })
+
+        const expectedDeleteCount = confirmationResponse ? 1 : 0
+        assert.strictEqual(
+            deleteCount,
+            expectedDeleteCount,
+            `Expected delete count ${expectedDeleteCount}, actual count ${deleteCount}`
+        )
+
+        const expectedRefreshCount = 1
```

Comment:
Should we refresh even when the user declines the confirmation?

Reply:
That is a good question. I haven't participated in a discussion about when we want to refresh the list of Lambdas but I'd be fine either way.

Reply:
It's probably better not to refresh when the user doesn't confirm since it might cause the Lambdas displayed to shift which might confuse the user who is not expecting a change. Fixed to behave this way.